demo/lib/vm.bash

```diff
@@ -174,6 +174,49 @@ vm-check-running-binary() {
     return 0
 }
 
+vm-wait-process-startup() {
```

Comment:
`vm-wait-process-startup [--timeout X]` seems to do the same as `vm-wait-process [--timeout X]`. What do you think, would it make sense to add `--pidfile` directly to `vm-wait-process` instead of adding a wrapper with the parameter?

But if there are reasons to have two separate functions, this is ok, too.

Reply:
Indeed, those two could be combined as the only difference is the pidfile option. I will change this.